README.md

```diff
@@ -1 +1,170 @@
-# hello-react-front-end
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+<h1><b>hello-react-front-end</b></h1>
+ 
+  <img src="./page1.png" alt="logo" width="200"  height="auto">
+  <img src="./page2.png" alt="logo" width="200"  height="auto">
+  <img src="./erd_diagram.png" alt="logo" width="200"  height="auto">
+  <br/>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [hello-react-front-end] <a name="about-project"></a>
+
+**[hello-react-front-end]**  is a web application that serves as a test for the Microverse Final Capstone project, which teaches how to use react app as a front end of a rails app backend.
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
it would be better to update the name of this section to Link to backend repository as your prohect curently does not have a deployed link please update it 👍🏿